src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
 public class WorkWithFile {
+    static final String COMMA_SPLITTER = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> fileData = readDataFromFile(fromFileName);
+
+        String report = createReport(fileData);
+
+        writeDataToFile(toFileName, report.getBytes());
```

Comment:
```suggestion
        List<String> fileData = readDataFromFile(fromFileName);
        String report = createReport(fileData);
        writeDataToFile(toFileName, report.getBytes());
```

lets remove redundant empty lines